### DISCUSSION TITLE
CLI showing usage text on config file errors

### DISCUSSION THREAD

**Discussion by daonb** [ngrok/ngrok#18] [Bug Reports]
I've been playing with my config file and using `ngrok config check` to check my work.  The attached screen shot ends with the two ERROR lines that really matter and above them over 20 lines of irrelevant output:
     
![IMG_2725](https://github.com/user-attachments/assets/cd17feff-499e-4987-9fed-029db1788612)

**Comment by russorat** [comment]
@daonb thanks for this. i chatted with the team and they agree showing the help text here isn't needed. I will open an issue to get this fixed in the next agent version.

**Reply by russorat** [reply]
created https://github.com/ngrok/ngrok/issues/19 for this
